src/Microsoft.AspNetCore.Http.Connections.Client/Internal/ServerSentEventsTransport.cs

```diff
@@ -72,6 +74,13 @@ public ServerSentEventsTransport(HttpClient httpClient, ILoggerFactory loggerFac
                 throw;
             }
 
+            // Create the pipe pair (Application's writer is connected to Transport's reader, and vice versa)
+            var options = ClientPipeOptions.DefaultOptions;
+            var pair = DuplexPipe.CreateConnectionPair(options, options);
+
+            _transport = pair.Application;
+            _application = pair.Transport;
```

Comment:
I know the names of the ones on DuplexPipe don't really matter, but can you make them line up to avoid confusion?